### PR TITLE
NDPD-1254: Check for score before creating empty submission.

### PR DESCRIPTION
**Pull request by vkaracic**

### CODE REVIEW

File: edx_sga/sga.py [hunk 1]
```diff
@@ -563,6 +563,14 @@ def enter_grade(self, request, suffix=''):
         module_id = request.params.get('module_id')
         student_id = request.params.get('student_id')
 
+        if not score:
+            return Response(
+                json_body=self.validate_score_message(
+                    module.course_id,
+                    module.student.username
```
Comment (marjev) [inline edx_sga/sga.py]: I think module is not yet defined in this context. I think line 575 introduces it.

**Review by marjev — CHANGES_REQUESTED** [review]

**Comment by marjev** [comment]
Also, version bump

**Comment by vkaracic** [comment]
@marjev comments addressed!

**Review by marjev — APPROVED** [review]

**Comment by marjev** [comment]
Also, should we add the check on the front end side as well?

**Comment by vkaracic** [comment]
There were checks on the frontend but they didn't work because it turns out, and you'll love this, `Number('')` returns a `0` value. #programminglanguagegoals

**Comment by marjev** [comment]
image

image

**Pull request merged**